src/main/java/org/prebid/server/bidder/ix/IxBidder.java

```diff
@@ -132,6 +136,22 @@ private ExtImpIx parseImpExt(Imp imp) {
         return modifiedBidRequests;
     }
 
+    /**
+     * IndexExchange does nothing if `buyerid` is not present on the request.
+     * This methods adds the default `0` buyerid. Maybe IX will add the IX UID to this adapter in the future.
+     *
+     * @param user user object for the current bid request
+     * @return the user with a default value for buyerid
+     * @see [https://kb.indexexchange.com/publishers/openrtb_integration/list_of_supported_openrtb_bid_request_fields_for_sellers.htm#User]
+     */
```

Comment:
We prefer not to use comments in bidder code

Reply:
Is there a reason for that :smiley: ?